test/javascript/components/mentoring/DiscussionPostForm.test.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import { fireEvent, render, waitFor } from '@testing-library/react'
+import { rest } from 'msw'
+import { setupServer } from 'msw/node'
+import '@testing-library/jest-dom/extend-expect'
+import { DiscussionPostForm } from '../../../../app/javascript/components/mentoring/DiscussionPostForm'
+
+test('shows loading message', async () => {
```

Comment:
Maybe expand a bit on this?

```suggestion
test('shows loading message while fetching posts', async () => {
```